Algorithm/QCAlgorithm.cs

```diff
@@ -1672,6 +1668,60 @@ public Option AddFutureOptionContract(Symbol symbol, Resolution? resolution = nu
             return AddOptionContract(symbol, resolution, fillDataForward, leverage);
         }
 
+        /// <summary>
+        /// Creates and adds index options to the algorithm.
+        /// </summary>
+        /// <param name="ticker">The ticker of the Index Option</param>
+        /// <param name="resolution">Resolution of the index option contracts, i.e. the granularity of the data</param>
+        /// <param name="fillDataForward">If true, this will fill in missing data points with the previous data point</param>
+        /// <param name="leverage">The leverage to apply to the index option contracts</param>
+        /// <returns>Canonical Option security</returns>
+        public Option AddIndexOption(string ticker, Resolution? resolution = null, string market = Market.USA, bool fillDataForward = true, decimal leverage = Security.NullLeverage)
```

Comment:
Not sure Leverage applies here, think we can lose the argument really? Same for the other new methods